### PR TITLE
Liquibase changes for pepper-68 and pepper-900

### PR DESCRIPTION
**Pull request by denniscunningham**
pepper-68
pepper-900

The changeset PEPPER-68_location_px.xml increases the size of ddp_onc_history_detail.location_px to 1000 characters.
The changeset PEPPER-900_field_settings.xml adds support for a controlled vocabulary for onc history result status.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [ ] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.
- [ ] If applicable, I have discussed the analytics needs at both a platform and study level with Product and instrumented code accordingly.
- [ ] If applicable, my UI/UX changes have passed muster with Product/Design via an over-the-shoulder review, screenshots, etc.

_If unsure or need help with any of the above items, add the `help wanted` label. For items that starts with `If applicable`, if it is not applicable, check it off and add `n/a` in front._

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [x] They are user-visible in dev as a regular user journey and require no additional instructions.
- [ ] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [ ] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [x] These changes require no special release procedures--just code!
- [ ] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-900_field_settings.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+    <changeSet author="cunningh" id="PEPPER-900_field_settings">
+        <insert tableName="field_settings">
+            <column name="ddp_instance_id"
+                    valueComputed="(SELECT ddp_instance_id FROM ddp_instance WHERE instance_name='cmi-lms')"></column>
+            <column name="field_type" value="VOCABULARY"/>
+            <column name="column_name" value="REQUEST_STATUS"/>
+            <column name="display_type" value="OPTIONS"/>
+            <column name="possible_values"
+                    value='[{"value": "Needs Review"},{"value": "Don&apos;t Request"},{"value": "On Hold"},{"value": "Request"},{"value": "Sent"},{"value": "Received"},{"value": "Returned"},{"value": "Unable to Obtain"}]'/>
```
Comment (otchet-broad) [inline pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-900_field_settings.xml:16]: Is that the github UI messing with the apostrophe should this be an escaped apostrophe: 
'[{"value": "Needs Review"},{"value": "Don\'t Request"},{"value": "On Hold"},{"value": "Request"},{"value": "Sent"},{"value": "Received"},{"value": "Returned"},{"value": "Unable to Obtain"}]'

Comment (otchet-broad) [reply pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-900_field_settings.xml:16]: Oh for the love of....
It removes my escape and renders in this UI.

Comment (otchet-broad) [reply pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-900_field_settings.xml:16]: Screenshot 2023-06-16 at 10 55 58 AM

Comment (denniscunningham) [reply pepper-apis/dsm-core/src/main/resources/liquibase/PEPPER-900_field_settings.xml:16]: Liquibase got the apostrophe right when I ran the script locally.

**Review by otchet-broad — APPROVED** [review]

**Pull request merged**